pandas/tests/io/excel/test_readers.py

```diff
@@ -57,6 +57,13 @@ def ignore_xlrd_time_clock_warning():
                 pytest.mark.filterwarnings("ignore:.*(tree\\.iter|html argument)"),
             ],
         ),
+        pytest.param(
+            "pyxlsb",
+            marks=[
+                td.skip_if_no("pyxlsb"),
+                pytest.mark.filterwarnings("ignore:.*(tree\\.iter|html argument)"),
```

Comment:
Is filterwarnings actually required here?

Reply:
Was seeing some defusedxml warnings but they're present either way, so I've removed filterwarnings.